Source/towners.cpp

```diff
@@ -47,6 +47,7 @@ void InitTownerInfo(int16_t i, const TownerData &townerData)
 	auto &towner = Towners[i];
 
 	towner._ttype = townerData.type;
+	towner.name = TownerLongNames[townerData.type];

```

Comment:
I think you are missing the actual translation call here.
```suggestion
	towner.name = _(TownerLongNames[townerData.type]);

```

Reply:
Thanks, fixed!